docs/REST-API/20-Alert-Grouping-Settings-API.md

```diff
@@ -0,0 +1,35 @@
+---
+tags: [rest-api]
+---
+
+# Alert Grouping Settings API - FAQ 
+
+
+### What is the maximum number of services for a Global Alert Grouping setting?
+
+With Global Alert Grouping - Content Based, there will be a **limit of 250 services maximum per alert grouping setting.**
+
+
+### What is the difference between the [Services API] endpoints where I set the alert grouping parameters for a given service vs. the [Alert Grouping Settings API] endpoints?
+
+- Existing `/services` endpoints only allow you to define Single Service Alert Grouping.
+- The new `/alert_grouping_settings` endpoints allow you to define Single or Global Content Based Alert Grouping.
+- Single Service Time and Single Intelligent Grouping types can still be set using the `/services` endpoints.
+
+
+### When can I use only the new [Alert Grouping Settings API] endpoints?
+
+In the future, you will be able to set **all** alert grouping via the `/alert_grouping_settings` endpoints, including: 
+- Create or update Single or Multi-Service (Global) Alert Grouping settings for all types (e.g., `content_based_intelligent` alert grouping type with a flexible time window).
```

Comment:
1. I don't think we've made any announcement wrt `content_based_intelligent` so let's use a different type maybe?
2. This seems to imply you can't use the API for all grouping types (`time` and `intelligent` as well), but you can. I'm not totally sure what it's trying to communicate? Is it regarding `global` settings for other types?

Reply:
I removed `content_based_intelligent` as you mentioned, and reworded this to capture that this new endpoint will be the only way to set alert grouping settings. Hopefully this is clear.